linux_os/guide/system/auditing/authconfig_config_files_symlinks/oval/shared.xml

```diff
@@ -0,0 +1,48 @@
+<def-group>
+    <definition class="compliance" id="{{{ rule_id }}}" version="1">
+        {{{ oval_metadata("Prevent overwriting of custom authentication configuration settings by the authconfig utility") }}}
```

Comment:
I feel the OVAL description should be more aligned to the message in the "rule title". What do you think?

Reply:
I used the rule title as the description here. I'm open to suggestions on a better wording.

Reply:
Thanks @freddieRv . It is better now IMO.